Readability.js

```diff
@@ -2029,12 +2120,15 @@ Readability.prototype = {
     // Unwrap image from noscript
     this._unwrapNoscriptImages(this._doc);
 
+    // Extract JSON-LD metadata before removing scripts
+    var jsonLd = this._preferJSONLD ? this._getJSONLD(this._doc) : {};
```

Comment:
Security: Will `<script>` elements be available here, when Readability is integrated in Firefox?

Reply:
Yes, they're only removed on the next line, when the script tags are all removed. We use JSDOMParser in a worker, and so the DOM is inert because JSDOMParser doesn't run script at all. I forget exactly how JSDOMParser treats script internals, but if the tests you have that rely on the contents of those tags pass in node.js, they should work in Firefox, too.